README.md

```diff
@@ -1,2 +1,15 @@
-pulp_packaging
+Pulp packaging
```

Comment:
Since it's a title, I'd prefer it to be

Pulp Packaging
===========
instead of
Pulp packaging
============